hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/SqlSelectTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.map;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.sql.SqlTestSupport;
+import com.hazelcast.map.IMap;
+import com.hazelcast.test.annotation.ParallelJVMTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.junit.Assert.assertEquals;
+
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class SqlSelectTest extends SqlTestSupport {
+    @BeforeClass
+    public static void setUpClass() {
+        initialize(1, null);
+    }
+
+    private static List<Row> fillIMapAndGetData(IMap<Integer, String> map, final int count) {
+        assert count >= 0;
+        List<Row> rows = new ArrayList<>(count);
+        int i;
+        for (i = 0; i < count; ++i) {
+            char ch = (char) (65 + i);
+            String s = String.valueOf(ch);
+            map.put(i, s);
+            rows.add(new Row(i, s));
+        }
+        assertEquals(i, count);
+        return rows;
+    }
+
+    @Test
+    @Ignore
```

Comment:
All tests are ignored? They run fine locally for me.

Reply:
They are, because IMDG engine was used 😄 
But you are right, no sense to ignore them.